multiqc/multiqc.py

```diff
@@ -245,6 +245,12 @@
     is_flag=True,
     help="Don't upload generated report to MegaQC, even if MegaQC options are found",
 )
+@click.option(
+    "--no-versions-section",
+    "no_versions_section",
+    is_flag=True,
+    help="Exclude section 'Software Versions' listing software versions from the report",
+)
```

Comment:
I think this was added because of https://github.com/ewels/MultiQC/pull/1927#discussion_r1310019204 - I didn't really mean a new CLI flag. I'm not sure that this would be used enough to warrant another CLI flag? The list is already pretty long, I try to only add things when needed.

Additionally: when adding a new CLI flag, it should be configured for `rich.click` in the config above so that it gets grouped properly.

Reply:
Yes a CLI option is definitely unnecessary here, I think I got confussed in you previoud comment 😉 I will remove the `--no-versions-verions` flag.

Reply:
Fixed in 49647e1eb8816a8b2d245b04b3ccd0e9ca3b50fb

Reply:
Thank you, and sorry for not being clearer originally! 🌟